flixnchill/CardsViewController.m

```diff
@@ -91,13 +105,84 @@ - (IBAction)onInfoButtonTapped:(UITapGestureRecognizer *)sender {
 
 - (IBAction)onLikeTapped:(UITapGestureRecognizer *)sender {
     NSLog(@"CardsViewController:Like tapped");
+    [self showBlur];
+    
+    [self.movieCardsView setUpView:self.movies withDraggableImageView:self.draggableCard];
+    [self.movieCardsView setHidden:NO];
+    [self.view bringSubviewToFront:self.movieCardsView];
+    
     [self.draggableCard swipeRight];
 }
 
 #pragma mark - END Gesture recognizers
 
+- (ThreeMoviesView*)setUpMoviesView {
+    ThreeMoviesView *threeMoviesView = [[[NSBundle mainBundle] loadNibNamed:@"ThreeMoviesView" owner:self options:nil] objectAtIndex:0];
+    threeMoviesView.center = CGPointMake(160.0f, 310.0f);
+    threeMoviesView.delegate = self;
+    [threeMoviesView setHidden:YES];
+
+    return threeMoviesView;
+}
+
+- (void)hideBlur {
```

Comment:
Nice Pattern here. For this couple of methods , Can you make this take in the subView to show / hide ?